### ISSUE TITLE
incompatible character encodings: ASCII-8BIT and UTF-8

### ISSUE THREAD

**Issue by wyuenho** [senchalabs/jsduck#491]
I'm not sure if this is an rkelly bug or jsduck, but I'm trying to build backgrid's doc and I have this [ES5 shim code](https://github.com/wyuenho/backgrid/blob/master/src/preamble.js#L13) embedded into my source file.

I don't understand why ASCII-8BIT and UTF-8 are not compatible.

I get this error in my console:

```
$ jsduck --output api src/
Error: Error while parsing /Users/wyuenho/Documents/workspace/backgrid/src/preamble.js: incompatible character encodings: ASCII-8BIT and UTF-8

Here's a full backtrace:
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:480:in `gsub'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:480:in `string_value'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:68:in `adapt_node'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:101:in `adapt_node'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:101:in `adapt_node'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:47:in `adapt_node'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:387:in `adapt_node'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:371:in `block in adapt_node'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:371:in `map'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:371:in `adapt_node'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:49:in `block in adapt_node'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:49:in `map'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:49:in `adapt_node'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:35:in `adapt_root'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/rkelly_adapter.rb:9:in `adapt'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/js/parser.rb:27:in `parse'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/parser.rb:44:in `parse_js_or_css'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/parser.rb:30:in `parse'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/batch_parser.rb:25:in `block in parse'
/Users/wyuenho/.gem/ruby/2.0.0/gems/parallel-0.7.1/lib/parallel.rb:339:in `call'
/Users/wyuenho/.gem/ruby/2.0.0/gems/parallel-0.7.1/lib/parallel.rb:339:in `call_with_index'
/Users/wyuenho/.gem/ruby/2.0.0/gems/parallel-0.7.1/lib/parallel.rb:268:in `process_incoming_jobs'
/Users/wyuenho/.gem/ruby/2.0.0/gems/parallel-0.7.1/lib/parallel.rb:251:in `block in worker'
/Users/wyuenho/.gem/ruby/2.0.0/gems/parallel-0.7.1/lib/parallel.rb:244:in `fork'
/Users/wyuenho/.gem/ruby/2.0.0/gems/parallel-0.7.1/lib/parallel.rb:244:in `worker'
/Users/wyuenho/.gem/ruby/2.0.0/gems/parallel-0.7.1/lib/parallel.rb:232:in `block in create_workers'
/Users/wyuenho/.gem/ruby/2.0.0/gems/parallel-0.7.1/lib/parallel.rb:231:in `each'
/Users/wyuenho/.gem/ruby/2.0.0/gems/parallel-0.7.1/lib/parallel.rb:231:in `create_workers'
/Users/wyuenho/.gem/ruby/2.0.0/gems/parallel-0.7.1/lib/parallel.rb:195:in `work_in_processes'
/Users/wyuenho/.gem/ruby/2.0.0/gems/parallel-0.7.1/lib/parallel.rb:106:in `map'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/util/parallel.rb:28:in `map'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/batch_parser.rb:17:in `parse'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/app.rb:41:in `parse'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/lib/jsduck/app.rb:21:in `run'
/Users/wyuenho/.gem/ruby/2.0.0/gems/jsduck-5.3.4/bin/jsduck:25:in `<top (required)>'
/Users/wyuenho/.gem/ruby/2.0.0/bin/jsduck:23:in `load'
/Users/wyuenho/.gem/ruby/2.0.0/bin/jsduck:23:in `<main>'

```

**Comment by wyuenho** [comment]
@nene This is still a problem in the latest 5.x branch.

**Comment by wyuenho** [comment]
@nene This was fixed 2.5 years ago and yet 6.x still hasn't been released. Mind releasing 5.3.5 with this fix included?
